Tests/AppTests/Common/Extensions/Foundation/Double/Double+RandomValue.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension Double {
+  /// Random double value,
+  /// rounded to 4 places, for ex: 1.212233244 -> 1.2122
+  static var randomValue: Double {
+    let r = random(min: 1.111, max: 333.333)
+    let divisor = pow(10.0, 4.0)
+    return Darwin.round(r * divisor) / divisor
```

Comment:
Is it work on linux?  I guess Darwin is absent on linux

Reply:
Fixed